pages/games/[hash].js

```diff
@@ -1,34 +1,80 @@
-import styles from "../../styles/Game.module.css";
+import { useState } from "react";
 
 function Game({ data }) {
+  const PLAYER = "PLAYER";
+  const OPPONENT = "OPPONENT";
+
+  const [visibleBoard, setVisibleBoard] = useState(PLAYER);
+
+  function toggleBoard() {
+    if (visibleBoard == PLAYER) {
+      setVisibleBoard(OPPONENT);
+    } else {
+      setVisibleBoard(PLAYER);
+    }
+  }
+
   return (
-    <div>
-      Game page {data.game_hash}
-      <div className={styles.board}>
-        <div className={styles.fleet}>
-          {data.ships.map((ship, i) => (
-            <div
-              key={ship.id}
-              className={styles.square}
-              style={formatGridAttributes(ship.grid_attributes)}
-            >
-              <div className={styles.ship}>
-              </div>
+    <main className="w-screen p-5 flex flex-col">
+      <h1 className="text-2xl">Game page {data.game_hash}</h1>
+
+      <div className="flex items-center justify-between my-2">
+        {visibleBoard == PLAYER ? <h2>Ma flotte</h2>:<h2>Flotte Adverse</h2>}
```

Comment:
You need a better linter. The best solution is to install prettier and eslint on your project and commit the config.

Reply:
What is the problem here?

Reply:
Missing space, but more importantly, you never use `==` in js you use strict equality `===` instead.
The problem with shallow equality (`==`) is that js will try to convert the value to match.
So 
```js
false == 0; // true
[] == false; // true
'' == false; // true
'' == [] // true
```
There are more some of which are even more wtf.